ui/raidboss/data/05-shb/dungeon/akadaemia_anyder.js

```diff
@@ -266,9 +266,10 @@
         'Eden Prime': 'Eden Prime',
         'Doliodus': 'Doliodus',
         'Cladoselache': 'Cladoselache',
-        'Ichthyology': 'Ichthyologie',
-        'Phytobiology': 'Phytobiologie',
-        'Phantomology': 'Phantomologie',
+        'Ichthyology will be sealed off': 'Noch 15 Sekunden bis sich der Zugang zu Ichthyologie schließt',
+        'Phytobiology will be sealed off': 'Noch 15 Sekunden bis sich der Zugang zu Phytobiologie schließt',
+        'Phantomology will be sealed off': 'Noch 15 Sekunden bis sich der Zugang zu Phantomologie schließt',
```

Comment:
Noch 15 Sekunden, bis sich die Ichthyologie schließt.
Noch 15 Sekunden, bis sich die Phytobiologie schließt.
Noch 15 Sekunden, bis sich die Phantomologie schließt.

These are the 3 lines which i gathered from a saved log